tests/providers/mysql/operators/test_mysql.py

```diff
@@ -139,7 +141,7 @@ class MySqlHookForTests(MySqlHook):
 
     dbapi_hook = MySqlHookForTests()
 
-    class SQLExecuteQueryOperatorForTest(MySqlOperator):
+    class SQLExecuteQueryOperatorForTest(SQLExecuteQueryOperator):
         def get_db_hook(self):
             return dbapi_hook
```

Comment:
i am not sure if this is the right way to test?
The `SQLExecuteQueryOperatorForTest` feels strange, we should test the actual operator I don't see a reason why we need to have extra class on top of it for testing

I see we do things differently in Postgres
https://github.com/apache/airflow/blob/e8af807cad1eafb52a5e94c1da15cdff2027696c/tests/providers/postgres/operators/test_postgres.py#L140

Reply:
Similar test is covered in other test case `test_mysql_operator_openlineage`. One addition in this test cases they are comparing the schema.

Reply:
But why do we need the custom class to achieve that?

Reply:
Not required. However, the test case is written in that way to inject the mocked dbapi_hook. 
I have updated it without additional class.